### PR TITLE
Anchore Grype: Default to Info severity if missing in report

### PR DESCRIPTION
**Pull request by manuel-sommer**
see #9618

### CODE REVIEW

**Review by mtesauro — APPROVED** [review]
Approved

File: unittests/scans/anchore_grype/issue_9618.json
Comment (Maffooch) [inline unittests/scans/anchore_grype/issue_9618.json:1]: Please reduce the size of this file

**Review by Maffooch — CHANGES_REQUESTED** [review]

**Comment by manuel-sommer** [comment]
Done @Maffooch, please rereview. :-)

**Review by cneill — APPROVED** [review]

**Comment by manuel-sommer** [comment]
@Maffooch I have shrink the file with about 50% https://github.com/DefectDojo/django-DefectDojo/pull/9620/commits/fd6197f824cc941ecb4ac001a7d7f8b05e7a22ab

**Comment by Maffooch** [comment]
I apologize @manuel-sommer I should have been more specific, can you please reduce the number of findings produced to something more reasonable like 10 findings? Having 240 findings to be parsed for a single test increases the time unit tests will run, and increases the size of the repo without much gain

**Comment by manuel-sommer** [comment]
Done @Maffooch. Please review again.

**Review by Maffooch — APPROVED** [review]

**Review by devGregA — APPROVED** [review]

**Pull request merged**